cromwell.example.backends/PAPIv2.conf

```diff
@@ -28,6 +28,11 @@ backend {
         # Emit a warning if jobs last longer than this amount of time. This might indicate that something got stuck in PAPI.
         slow-job-warning-time: 24 hours
 
+        # Whether to allow users to specify the 'noAddress' runtime attribute when submitting jobs.
+        # Defaults to true.
+        # If set to false, the noAddress attribute will be ignored and public addresses will always be assigned.
+        allow-noAddress-attribute: true
```

Comment:
mixed casing with kebabs? 🤢

Reply:
I'm by no means totally sure this is right, but it was kind of deliberate. The different casing styles indicate that the `noAddress` is a different level of mashing-together than the `allow-...-attribute` mashing.

Reply:
kind of like using `''`s inside `""`s

Reply:
alternative suggestions welcome!

Reply:
maybe `allow-private-ips`